test/test-pages.js

```diff
@@ -2,13 +2,6 @@ const expect = require('chai').expect;
 const request = require('request');
 require('../app');
 
-it('Landing page content', function (done) {
-  request(process.env.BASE_URL, function (error, response, body) {
-    expect(body).to.contain('Find the best organizations to donate your items to');
-    done();
```

Comment:
Can we update this instead of removing it?

Reply:
Done 👍